tests/lib/prepare-restore.sh

```diff
@@ -141,6 +141,46 @@ build_rpm() {
     fi
 }
 
+build_arch_pkg() {
+    base_version="$(head -1 debian/changelog | awk -F '[()]' '{print $2}')"
+    version="1337.$base_version"
+    packaging_path=packaging/arch
+    archive_name=snapd-$version.tar
+
+    rm -rf /tmp/pkg
+    mkdir -p "/tmp/pkg/sources/snapd"
+    cp -ra -- * "/tmp/pkg/sources/snapd/"
+
+    # shellcheck disable=SC2086
+    (tar -C /tmp/pkg/sources -cf "/tmp/pkg/$archive_name" "snapd")
+    cp "$packaging_path"/* "/tmp/pkg"
+
+    # fixup PKGBUILD which builds a package named snapd-git with dynamic version
+    #  - update pkgname to use snapd
+    #  - kill dynamic version
+    #  - packaging functions are named package_<pkgname>(), update it to package_snapd()
+    #  - update source path to point to local archive instead of git
+    #  - fix package version to $version
+    sed -i \
+        -e "s/^source=.*/source=(\"$archive_name\")/" \
+        -e "s/pkgname=snapd.*/pkgname=snapd/" \
+        -e "s/pkgver=.*/pkgver=$version/" \
+        -e "s/package_snapd-git()/package_snapd()/" \
+        /tmp/pkg/PKGBUILD
+    awk '
+    /BEGIN/ { strip = 0; last = 0 }
```

Comment:
Can you please move one of the comments from above (or add a new one) that describes what is going on here.